### PR TITLE
Initial implementation of VGG 19

### PR DESCRIPTION
**Pull request by eari100**

### CODE REVIEW

File: bonsai/models/vgg19/tests/run_model.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+model = params.create_vgg19_from_pretrained(task_weights_path)
+
+# 3. Prepare dummy input
+batch_size = 8
+image_size = 224
+dummy_input = jnp.ones((batch_size, image_size, image_size, 3), dtype=jnp.float32)
+
+# 4. Warmup + profiling
+# Warmup (triggers compilation)
+_ = model_lib.forward(model, dummy_input)
```
[the remark this answers is not in the document]
Comment (eari100) [reply bonsai/models/vgg19/tests/run_model.py]: I've refactored the code to pass GraphDef and GraphState as separate parameters.

File: bonsai/models/vgg19/tests/run_model.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Copyright 2025 The JAX Authors.
```
[the remark this answers is not in the document]
Comment (eari100) [reply bonsai/models/vgg19/tests/run_model.py:1]: I've added the validation code for ResNet-50 directly, without any modifications.

[the remark this answers is not in the document]
Comment (eari100) [reply bonsai/models/vgg19/tests/run_model.py:1]: image

Would it be alright to use this example where the model struggles to predict a hammerhead?

[the remark this answers is not in the document]
Comment (jenriver) [reply bonsai/models/vgg19/tests/run_model.py:1]: Thanks, I'd say this is somewhat expected. VGG19's Top-5 Accuracy is ~90.0% (with Top-1 Accuracy ~71.1%), so this can be within the expected range.

I've tested this model against other images in https://github.com/EliSchwartz/imagenet-sample-images and the results looked fine.

**Review by jenriver — DISMISSED** [review]
Thanks for your contribution!

**Review by jenriver — APPROVED** [review]
Thanks for the contribution! 🚀
